build.gradle

```diff
@@ -39,24 +46,17 @@ def depVersions = [
 
 ]
 
-allprojects {
-    group = 'com.commercetools'
-    version = '2.2.0-dev' // this version is used in shadowJar.manifest.Implementation-Version and then at runtime as getImplementationVersion()
-
-    tasks.withType(JavaCompile) {
-        options.compilerArgs << "-Xlint:unchecked" << "-Xlint:deprecation"
-    }
-}
```

Comment:
moved to [`subprojects`](https://github.com/commercetools/commercetools-payone-integration/pull/190/files#diff-c197962302397baf3a4cc36463dce5eaR49) section

Reply:
why?

Reply:
because only subprojects are actually Java projects, the parent project is not just Java project, it is a kind of aggregator.